includes/models/model.llms.question.php

```diff
@@ -175,6 +176,29 @@ protected function get_creation_args( $args = null ) {
 
 	}
 
+
```

Comment:
Instead of adding this logic by overriding the function, can we use the default property values to automatically fallback here?

From the `___get()` method:

https://github.com/gocodebox/lifterlms/blob/1d61d0b2f54b3ce5578f796ac611a04f66d2d229/includes/abstracts/abstract.llms.post.model.php#L437

Then define the default value in the default properties array:

https://github.com/gocodebox/lifterlms/blob/master/includes/abstracts/abstract.llms.post.model.php#L112

Here's a real example of setting default properties: https://github.com/gocodebox/lifterlms/blob/237bf80035a77bacf36ca4d8e99a8f3df92407d2/includes/models/model.llms.lesson.php#L72-L82